src/PointwiseFunctions/AnalyticSolutions/Poisson/Lorentzian.hpp

```diff
@@ -0,0 +1,75 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "Elliptic/Systems/Poisson/Tags.hpp"    // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Poisson {
+namespace Solutions {
+
+/*!
+ * \brief A Lorentzian solution to the Poisson equation
+ *
+ * \details This implements the Lorentzian solution
+ * \f$u(\boldsymbol{x})=\left(1+r^2\right)^{-\frac{1}{2}}\f$ to the
+ * three-dimensional Poisson equation
+ * \f$-\Delta u(\boldsymbol{x})=f(\boldsymbol{x})\f$, where
+ * \f$r^2=x^2+y^2+z^2\f$. The corresponding source is
+ * \f$f(\boldsymbol{x})=3\left(1+r^2\right)^{-\frac{5}{2}}\f$.
+ *
+ * \note Corresponding 1D and 2D solutions are not implemented yet.
+ */
+template <size_t Dim>
+class Lorentzian {
+ public:
+  using options = tmpl::list<>;
+  static constexpr OptionString help{
+      "A Lorentzian solution to the Poisson equation."};
+
+  Lorentzian() = default;
+  Lorentzian(const Lorentzian&) noexcept = delete;
+  Lorentzian& operator=(const Lorentzian&) noexcept = delete;
+  Lorentzian(Lorentzian&&) noexcept = default;
+  Lorentzian& operator=(Lorentzian&&) noexcept = default;
+  ~Lorentzian() noexcept = default;
+
+  auto field_variables(const tnsr::I<DataVector, Dim>& x) const noexcept
```

Comment:
would there be any gain in making these static? This solution at least doesn't seem to depend on any stored parameters.

Reply:
That could work, it's in the last fixup. @wthrowe do you know of any subtleties that would make this being `static` incompatible to the non-static interface of the other analytic solutions?

Reply:
I didn't realize it was valid C++ to use static functions like that, but it is.  I don't know if there's any gain, but I don't see any harm to it either so I'm fine with the change.